apis/hive/v1/clusterpool_types.go

```diff
@@ -21,6 +21,12 @@ type ClusterPoolSpec struct {
 	// +required
 	Size int32 `json:"size"`
 
+	// RunningCount is the number of clusters we should keep running. The remainder will be kept hibernated until claimed.
+	// By default no clusters will be kept running (all will be hibernated).
```

Comment:
nit: can we include what the default value is in the api doc, or do you think most users will understand that 0 is the default value for interger

Reply:
Between the statement on L25 and the intuitiveness of "runningCount==0 means no running clusters" I think it's sufficient. MaxSize and MaxConcurrent also mention the default behavior, not the default value (though admittedly in those cases the default value is less intuitive).